### PR TITLE
Add tpvlaxmi-washingtondc

### PR DESCRIPTION
**Pull request by tpvlaxmi**
Added Favorite restaurant and favorite food.  

@githubteacher Is there any additional information, you want

### CODE REVIEW

**Comment by precisiondx** [comment]
Hello @tpvlaxmi What's the best place to eat :fish:? I really like :sushi: next time I visit DC

**Comment by tpvlaxmi** [comment]
You can have best :fish: dish at McCormick & Schmick's Seafood restaurant. I guess you like it :+1:

File: tpvlaxmi-washingtondc.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Washington, D.C
+
+### Places to Eat
+- Vapiano: Pizza, Pasta, Fresh herbs
```
Comment (tpvlaxmi) [inline tpvlaxmi-washingtondc.md:4]: What is the best pasta dish

File: tpvlaxmi-washingtondc.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+- Pitango: Gelato, Hot chocolate
+- Woodlands: Indian food
```
Comment (tpvlaxmi) [inline tpvlaxmi-washingtondc.md:6]: What is fav Indian dish

**Review by tpvlaxmi — COMMENTED** [review]
Some quick questions submitted

**Pull request merged**